src/job/controller.ts

```diff
@@ -39,32 +125,19 @@ async function getPlanningReleased(
   );
 }
 
-async function holdJob(req: express.Request, res: express.Response) {
+async function holdJob(_req: express.Request, res: express.Response) {
   ExpressHandler(
     async () => {
-      const { jobId } = req.body;
-
-      // Make sure jobId is provided
-      if (!jobId) {
-        return HTTPError('Please provide a jobId', 400);
-      }
-
-      // Find the job and if it doesnt exist, raise an error
-      const job = await JobModel.findById(jobId);
-
-      // Check if the job exists
-      if (!job) {
-        return HTTPError(`Job ${jobId} not found`, 404);
-      }
+      const { job } = res.locals;
 
       // If the job is not released, then do not hold
       if (!job.released) {
-        return HTTPError(`Job ${jobId} has not been released yet`, 405);
+        return HTTPError(`Job ${job._id} has not been released yet`, 405);
```

Comment:
We'll have to keep an eye on this. Some other identified will be more useful later on. the ID will mean nothing to an end user